Dockerfile.development

```diff
@@ -9,7 +9,7 @@ ENV MOJ_FILE_UPLOADER_ENDPOINT   replace_this_at_build_time
 ENV TAX_TRIBUNALS_DOWNLOADER_URL replace_this_at_build_time
 ENV SENTRY_DSN                   replace_this_at_build_time
 ENV GOVUK_NOTIFY_API_KEY         replace_this_at_build_time
-ENV GA_TRACKING_ID               replace_this_at_build_time
+ENV GA_TRACKING_ID               UA-92448719-1
```

Comment:
You needed to do this on your local copy, to get the GA stuff to work for you, but I'd prefer not to check it in.
If we just stick to replace_this_at_build_time, it's a nice way to show people what env vars they need to think about for 'real' deployments.